go.mod

```diff
@@ -9,3 +9,11 @@ require (
 	github.com/mattn/go-sqlite3 v2.0.2+incompatible
```

Comment:
I am not very familar with go packages so a bit confused here – how would one install the package without the sqlite3 dependency? Doesn't this require the dependency even if you don't use the `pkg/storage/sqlite3` imports?

Reply:
See this PR over here: https://github.com/snowplow/snowplow-tracking-cli/pull/45

You need to inspect the compiled binary but Go prunes dependencies that are not needed which it can only do well if you use a modular package structure like what I have moved to here.